client/src/ui/OutputLogger.ts

```diff
@@ -0,0 +1,74 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) 2023 Savoir-faire Linux. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode'
+
+export class OutputLogger {
```

Comment:
From my experience so far, the winston logger sometimes didn't work (e.g. analyzer.ts). But the `connection.sendNotification() always works. 

If we only care about logs then having our custom logger in this case is ok IMO. Do we want to have something ful-featured like winston?

Reply:
All the extensions I have seen use custom loggers (cmake, docker, ...) which maybe is not ideal!
I needed a different logger because it's great to have separate output channels for the language server and the extension (as a language client). We can address this later for sure!